requirements/image.txt

```diff
@@ -1,4 +1,4 @@
 scipy
-torchvision
+torchvision==0.14.1
```

Comment:
We do not want one specific version so you say this is the min version?

Reply:
This was added by executing "make test". I should have excluded the requirements/image.txt and .gitignore changes. Sorry about that.

Reply:
However, what do you think about including the notebooks/ directory in the .gitignore file?

Reply:
```suggestion
torchvision
```